package.json

```diff
@@ -212,6 +213,9 @@
     "node": ">=7",
     "yarn": "^1.3"
   },
+  "browser": {
+    "child_process": false
```

Comment:
Since this package also works as is without the electron app, there are some dependencies which are needed. When we install the package on top of LBRY as a dependency, the internal dependencies are added to the node modules of LBRY, which is the correct behaviour and also works. But for some reason node.js things he needs to execute child processes inside the package and not globally on LBRY. Therefore, I added the parameter on package.json, to let node.js know he doesn't need to execute child_processes.

I wasn't sure if other packages on the app needs child_processes, but it doesn't seem so sinc webpack manages the whole settings of the different packages. Let me know if this could be a breaking change for the app.